connectors/sources/jira.py

```diff
@@ -56,11 +61,43 @@
     ISSUE_DATA: "/rest/api/2/issue/{id}",
     ATTACHMENT_CLOUD: "/rest/api/2/attachment/content/{attachment_id}",
     ATTACHMENT_SERVER: "/secure/attachment/{attachment_id}/{attachment_name}",
+    USERS: "/rest/api/3/users/search",
+    PROJECT_PERMISSIONS: "/rest/api/2/user/permission/search?projectKey={project_key}&permissions=BROWSE_PROJECTS",
+    PROJECT_ROLE_MEMBERS: "/rest/api/3/project/{project_key}/role/{role_id}",
+    ISSUE_SECURITY_LEVEL: "/rest/api/2/issue/{issue_key}?fields=security",
+    SECURITY_LEVEL_MEMBERS: "/rest/api/3/issuesecurityschemes/level/member?maxResults={max_results}&startAt={start_at}&levelId={level_id}&expand=user,group,projectRole",
 }
 
 JIRA_CLOUD = "jira_cloud"
 JIRA_SERVER = "jira_server"
 
+ACCESS_CONTROL = "_allow_access_control"
+ATLASSIAN = "atlassian"
+USER_QUERY = "expand=groups,applicationRoles"
+
+
+def _prefix_identity(prefix, identity):
+    if prefix is None or identity is None:
+        return None
+
+    return f"{prefix}:{identity}"
+
+
+def _prefix_username(user):
+    return _prefix_identity("username", user)
+
+
+def _prefix_account_id(user_id):
+    return _prefix_identity("account_id", user_id)
+
+
+def _prefix_group_id(group_id):
+    return _prefix_identity("group", group_id)
+
+
+def _prefix_role_key(role_key):
+    return _prefix_identity("application_role", role_key)
```

Comment:
(more like a note to myself): We should probably move some generic functionality into a separate `access_control.py` module to increase reusability. I think I've started that for the GMail connector before my holidays, so you can keep it here for this PR.